chain/chain/src/types.rs

```diff
@@ -171,7 +171,9 @@ impl BlockEconomicsConfig {
     const MAX_GAS_MULTIPLIER: u128 = 20;
     /// Compute min gas price according to protocol version and genesis protocol version.
     pub fn min_gas_price(&self, protocol_version: ProtocolVersion) -> Balance {
```

Comment:
We probably want to add a test here. I am not sure where existing tests are, but one way to figure that out would be to introduce a bug in this function and see what breaks.